src/test/java/de/rabea/player/UnbeatableComputerPlayerTest.java

```diff
@@ -88,4 +89,22 @@ public void blocksAnOpponentsWinWithFourMovesLeft() {
                                             EMPTY, EMPTY,O});
         assertEquals(1, unbeatableComputerPlayer.getPosition(board));
     }
+
+    @Ignore
```

Comment:
If you want to make sure you meet a certain threshold, you might find JUnit's `timeout` handy. https://github.com/junit-team/junit/wiki/Timeout-for-tests